pkg/cli/cmd/flags.go

```diff
@@ -14,4 +14,21 @@ var (
 		Value:       "/eib",
 		Destination: &BuildArgs.ConfigDir,
 	}
+	GenerateDefinitionFileFlag = &cli.StringFlag{
```

Comment:
In fact, why do we have to duplicate all these? Can't we reuse the exact same ones we already have?